### DISCUSSION TITLE
Welcome to mesoSPIM-control Discussions!

### DISCUSSION THREAD

**Discussion by ffvoigt** [mesoSPIM/mesoSPIM-control#53] [General]
## 👋 Welcome!
 
We’re using Discussions as a place to connect with other members of the mesoSPIM community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drchrisch** [comment]
Hi from mesoSPIM V5 setup at ISP, Hamburg.

Starting the *Discussions* section is a good idea.
